source/collections/LinkedList.ooc

```diff
@@ -153,12 +154,20 @@ operator []= <T>(list: LinkedList<T>, index: Int, value: T) { list set(index, va
 Node: class <T> {
 	prev: This<T>
 	next: This<T>
-	data: T* //TODO: Auto-wrap covers in Cell<T>
+	_data: Cell<T>
```

Comment:
Well, it's nice to be able to assume that if you have a `Cell`, you will never find an object inside. But I don't understand why you are wrapping everything in a `Cell` in the first place. I don't think we're doing that anywhere else...? `HashDictionary` does it, but only if `T` is a cover, not otherwise. And I don't see why any one `LinkedList` should be able to contain more than one type of `T` at the same time.